Add support for exporting bitwarden card, identity and secure notes

firecat53 opened the pull request:
Fixes one bug and as the title says, also will import secure notes, cards and
identities.

- adds a '- Card', '- Identity', '- Secure Note' to the title
- adds any extra URLs to notes for the login type
- adds card or identity info to the notes

firecat53 committed:
Fix issue with items in Root group or empty usernames

firecat53 committed:
Add support for bitwarden card, identity and secure note types

firecat53 committed:
Update README to reflect support of card/identity types

k3karthic commented:
Hi @firecat53,

Thank you for taking the time to contribute to the project. I've reviewed the code, and I am merging the pull request. Will release v0.1.4 with your changes over the weekend.

Regards,

firecat53 commented:
Thanks! I actually went through a restructed the project to make it easier to use. You can see it on my development branch (https://github.com/firecat53/bitwarden-to-keepass/tree/develop).

- Converted it to single executable file. No installation needed. It's a short and simple script, so needing to make it a package probably isn't necessary.
- Edited README
- Moved and renamed tests.py into root directory. Tests still work and pass with `python -m unittest`.

Definitely not trying to hijack your project! We all have different approaches.  I'm happy to submit my changes as a PR, or just leave it as a friendly fork on my account.

Scott

k3karthic commented:
I really like the changes you made and would be happy to integrate them if you submit a pull request. Thank you once again for taking the time to contribute.

If you're okay with it, I can add you as a collaborator to the project.

I've avoided making a pip package as I didn't think the script was good enough for end users. I'm happy to leave it as a script until  someone requests for it.
